docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Jakarta_Data.adoc

```diff
@@ -0,0 +1,52 @@
+[[Jakarta_Data]]
+= Jakarta Data Subsystem
+
+ifdef::env-github[]
+:tip-caption: :bulb:
+:note-caption: :information_source:
+:important-caption: :heavy_exclamation_mark:
+:caution-caption: :fire:
+:warning-caption: :warning:
+endif::[]
+
+[abstract]
+
+The `jakarta-data` subsystem provides support for the use of Jakarta Data in deployments.
+
+[WARNING]
+
+The `jakarta-data` subsystem is currently provided at xref:Admin_Guide.adoc#Feature_stability_levels[`preview` stability]. This means use of the subsystem requires running xref:WildFly_and_WildFly_Preview.adoc[WildFly Preview] or using the `--stability=preview` parameter when starting standard WildFly.
+
+[[jakarta-data-subsystem-provision]]
+== Provisioning the subsystem
+
+When provisioning a WildFly instance, you can include Jakarta Data support in your server configuration by specifying the `jakarta-data` Galleon layer.
+
+////
+TODO add discussion of the need to specify config-stability-level when provisioning. But this should point to general content available via WFLY-19021 and WFLY-19172
+////
+
+[[jakarta-data-subsystem-enable]]
+== Enabling the subsystem
+
+If the WildFly configuration does not have the `jakarta-data` subsystem enabled, it can be enabled using the CLI:
+
+[source,options="nowrap"]
+----
+[standalone@localhost:9990  /] org.wildfly.extension.jakarta.data:add
```

Comment:
From an external perspective it seems odd that adding the jakarta-data subsystem doesn't imply adding this one as well... ? Is there any reason someone would want one without the other?

Reply:
The "/subsystem=jakarta-data:add" command would fail if (the correct version of) this wasn't there.

The correct version being "/extension=org.wildfly.extension.jakarta.data:add". Which I'll fix.

An Extension can provide one or more subsystems. You have to add it before you can add its subsystems; until you do the server kernel has no clue what "subsystem=jakarta-data" means. The /extension=foo:add tells the server to load the 'foo' module and find the Extension interface impl in it. Which it calls to get the extension to register the code that allows that allows things like /subsystem=jakarta-data:add to work.

This is clunky but we've never had time to make it go away.

Reply:
I fixed the incorrect command. Thanks for asking!